src/Core/src/ImageSources/ImageSourceExtensions.cs

```diff
@@ -49,7 +49,8 @@ static async Task LoadImageResult(Task<IImageSourceServiceResult<PlatformImage>?
 				return Task.FromResult<IImageSourceServiceResult<PlatformImage>?>(null);
 
 #if IOS || MACCATALYST
-			return imageSourceService.GetImageAsync(imageSource);
+			var scale = mauiContext.GetPlatformWindow().Screen.Scale;
```

Comment:
Should we use the `GetDisplayDensity()` extension method, then we can use it in all the platforms.